extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -262,12 +254,13 @@ private static QuarkusPersistenceUnitDescriptor generateReactivePersistenceUnit(
             BuildProducer<HotDeploymentWatchedFileBuildItem> hotDeploymentWatchedFiles,
             List<DatabaseKindDialectBuildItem> dbKindDialectBuildItems) {
         //we have no persistence.xml so we will create a default one
-        String persistenceUnitConfigName = DEFAULT_PERSISTENCE_UNIT_NAME;
+        String persistenceUnitConfigName = REACTIVE_PERSISTENCE_UNIT_NAME;
```

Comment:
The config name must probably still be `<default>`. It's not the same as the PU name. See https://github.com/quarkusio/quarkus/commit/492a2d75174fc80ffadedec8787eff1e4ecb8cac

Reply:
Reverted to `DEFAULT_PERSISTENCE_UNIT_NAME`